documentation/content/docs/docker.md

```diff
@@ -184,3 +184,38 @@ sudo docker run --runtime=nvidia             \
 ```
 
 And you can access it by pointing your browser to `http://localhost:9000`
+
+Alternatively, the information needed to build the root of the session url can be provided as environment variables, rather than a command-line argument which follows the image tag.  In the examples above, you would remove the final argument looking like `"ws:<hostname>:<port>"` and instead set the following environment variables:
+
+```
+export SERVER_NAME="pvw.company.com"
+export PROTOCOL="ws"
+```
+
+Note that in the same way, extra arguments can be passed to the pvpython instances which are started: either as extra arguments after the session url in the docker run command, or as an environment variable named `EXTRA_PVPYTHON_ARGS`.
+
+Here's an example, in command line argument form, which sets the session url root to `ws://localhost:9000`, disables use of the ParaView registry, and chooses a specific mesa rendering backend:
+
+```
+sudo docker run --runtime=nvidia             \
+    -p 0.0.0.0:9000:80                        \
+    -v ~:/data                                 \
+    -ti kitware/paraviewweb:pvw-egl-demo-v5.6.0 \
+    "ws://localhost:9000/"
+    -dr --mesa-swr
+```
+
+And here's the same thing done using environment variables instead:
+
+```
+export SERVER_NAME="localhost:9000"
+export PROTOCOL="ws"
+export EXTRA_PVPYTHON_ARGS="-dr,--mesa-swr"
+
+sudo docker run --runtime=nvidia             \
+    -p 0.0.0.0:9000:80                        \
+    -v ~:/data                                 \
+    -ti kitware/paraviewweb:pvw-egl-demo-v5.6.0
```

Comment:
Should this be done with the `-e` argument so those env variables are indeed passed to docker?

```
sudo docker run --runtime=nvidia             \
    -p 0.0.0.0:9000:80                        \
    -v ~:/data                                 \
    -e SERVER_NAME="localhost:9000" \
    -e PROTOCOL="ws" \
    -e EXTRA_PVPYTHON_ARGS="-dr,--mesa-swr" \
    -ti kitware/paraviewweb:pvw-egl-demo-v5.6.0
```

Reply:
Yep, oversight on my part.